pygmt/src/grdinfo.py

```diff
@@ -0,0 +1,127 @@
+"""
+Retrieve info about grid file.
+"""
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    data_kind,
+    dummy_context,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    C="per_column",
+    D="tiles",
+    F="geographic",
+    I="spacing",
+    T="nearest_multiple",
+    L="force_scan",
+    M="minmax_pos",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(R="sequence", D="sequence", I="sequence")
```

Comment:
```suggestion
@kwargs_to_strings(D="sequence", I="sequence", R="sequence")
```
Alphabetical order here too.